automatic/1password/1password.nuspec

```diff
@@ -3,10 +3,10 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>1password</id>
-    <version>7.9.832</version>
+    <version>8.8.0</version>
```

Comment:
Do not update the version number used, you will prevent automatic updates from happening (at least for v8.8.0).

Reply:
Could you elaborate here? I don't understand what you mean.